docs/setup.asciidoc

```diff
@@ -15,6 +15,7 @@ On .NET Core the agent also supports auto instrumentation without any code chang
 * <<setup-sqlclient>>
 * <<setup-stackexchange-redis>>
 * <<setup-azure-servicebus>>
+* <<setup-mongo-db>>
```

Comment:
minor nit: we'll be adding more cloud integrations over time, so it might be better to put MongoDB integration before Azure ServiceBus, so that new cloud integrations are added after and grouped together.

Reply:
I merged in the other one and fixed conflicts, so this should be ok.